awesomemotive/affiliatewp-checkout-referrals
# Issue/65 - Bump WordPress minimum requirements

Issue #65


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate Checkout Referrals add-on
2. Confirm the plugin works as expected without any PHP errors.
3. Confirm existence and test saving the new settings on `Affiliates > Settings > Checkout Referrals`
